app/controllers/swagger_yard/swagger2_controller.rb

```diff
@@ -0,0 +1,9 @@
+module SwaggerYard
+  class Swagger2Controller < ApplicationController
```

Comment:
_If_ we want to support both spec versions, I would love to be able to use the same controller … and have it simply check the configured version. So maybe having the multiple classes respond to the same interface `Swagger1.new.to_h` and `Swagger2.new.to_h` or something?